scripts/compile-scss.js

```diff
@@ -74,6 +74,9 @@ async function compileScssFiles(
             error.stack
           }`
         );
+
+        process.exitCode = 1;
+        return;
```

Comment:
This return is a failfast mechanism to prevent the EUI aliased compilation from running and spitting out more errors